elide-integration-tests/src/test/java/com/yahoo/elide/tests/ResourceIT.java

```diff
@@ -737,8 +737,8 @@ public void testForbiddenDeleteEmptyCollectionRelationship() throws Exception {
 
     @Test(priority = 11)
     public void testDeleteParent() {
+        //Should work without content-type and body
         given()
```

Comment:
@Koka unfortunately, I cannot reproduce this behavior. This test works fine for me with the existing code even after removing `contentType`. Are you using Jersey? If not, perhaps this is an issue with specific servlet containers. Either way, we should fix it, but I am interested in reproducing it since this test (from my end) doesn't appear to be behaving as we expect.

Similarly, I ran the [blog example](https://github.com/yahoo/elide/tree/master/elide-example/elide-blog-example) and was able to execute these two queries with success:

```curl -H'Content-Type: application/vnd.api+json' \
               -H'Accept: application/vnd.api+json' --data '
          {
            "data": {
              "type": "user",
              "attributes": {
                "name": "Michael Admin",
                "role": "Admin"
              }
            }
          }
          ' -X POST http://localhost:4080/user
```

immediately followed by:

```
curl -X DELETE http://localhost:4080/user/2
```

Can you reproduce that behavior on your end? We should make sure we're reproducing this correctly so we don't break it in the future.